src/components/seek_time/seek_time.js

```diff
@@ -89,8 +90,17 @@ export default class SeekTime extends UIObject {
       this.$el.css('left', "-100%")
     }
     else {
-      var seekTime = this.hoverPosition * this.duration
-      var currentSeekTime = formatTime(seekTime)
+      if (this.actualLiveTime) {
+        var d = new Date(), e = new Date(d);
+        var secondsSinceMidnight = (e - d.setHours(0,0,0,0)) / 1000;
+        var seekTime = (secondsSinceMidnight - this.duration) + (this.hoverPosition * this.duration)
+        if (seekTime < 0) {
+          seekTime = (3600 * 24) + seekTime
+        }
+      } else {
+        var seekTime = this.hoverPosition * this.duration
+      }
```

Comment:
I totally agree on all your points. If you guys are okay with inline if statements I agree it will actually be a little cleaner. For the redundant formatActualTime(), I agree we could check for the actualLiveTime option inside formatTime() as there were only a little line change to get zero-filled hours. I will push some code later :)